<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.140704 rgba(126, 75, 117, 255), stop:0.442211 rgba(192, 97, 123, 255), stop:0.713568 rgba(241, 134, 111, 255), stop:1 rgba(255, 187, 99, 255));

color: #3b375b;
</string>
  </property>
  <widget class="QFrame" name="Frame1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>5000</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.140704 rgba(126, 75, 117, 255), stop:0.386935 rgba(192, 97, 123, 255), stop:0.628141 rgba(241, 134, 111, 255), stop:0.969849 rgba(255, 187, 99, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrgnImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExtractText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Extract Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="HiddenTextImage">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="SideFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>99999</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#3b375b;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>114</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:#0091a9;
color: #f3ecff;

font: bold;
padding: 6px;

min-width: 237%;
max-width: 100%;
}</string>
    </property>
    <property name="text">
     <string>WelcomeScreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Encode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>79</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:#0091a9;
color: #f3ecff;

font: bold;
padding: 6px;

min-width: 237%;
max-width: 100%;
}</string>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>128</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:#0091a9;
color: #f3ecff;

font: bold;
padding: 6px;

min-width: 237%;
max-width: 100%;
}</string>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
